test/verify/check-reauthorize

```diff
@@ -45,8 +45,11 @@ class TestReauthorize(MachineCase):
         b.click("#navbar-dropdown")
         b.click("#credentials-item")
         b.click("button.credential-clear")
-        b.wait_not_visible("button.credential-clear")
-        b.click("#credentials-dialog [data-dismiss]")
+        if self.machine.image in [ "rhel-8-1-distropkg" ]:
+            b.wait_not_visible("button.credential-clear")
+            b.click("#credentials-dialog [data-dismiss]")
+        else:
+            b.expect_load()
```

Comment:
I'd like to suggest that the test should do the following additional checks:
 * Open an arbitrary channel (eg: a connection to a unix socket), and verify that it has been closed when reloaded.
 * Check that that the ```--privileged``` process bridge has exited.

Reply:
I nade it so that the playground page holds a `flock` open, and we expect that to go away after the reload.  I'd say the test already checks for an exited priv bridge via the "Reauthorize" button.

Reply:
Nice. That's a good idea.